src/Database/Concerns/HasDomains.php

```diff
@@ -14,6 +14,9 @@
  */
 trait HasDomains
 {
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany<Domain, $this>
+     */
     public function domains()
```

Comment:
This also needs the return type


```suggestion
    public function domains(): HasMany
```